src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -69,11 +61,18 @@ def set_ansys_version(version: Union[str, float, FluentVersion]) -> None:
     and the environment variables are updated properly. This supersedes
     the Fluent path set in the environment variable.
     """
-    if type(version) in [float, str]:
-        version = FluentVersion.get_version(str(version))
-    if version in FluentVersion or str(version) in FluentVersion.value:
-        FLUENT_VERSION[0] = version.value
-        PIM_FLUENT_PRODUCT_VERSION[0] = FLUENT_VERSION[0].replace(".", "")
+    if isinstance(version, (float, str)):
+        version = str(version)
+    elif isinstance(version, FluentVersion):
+        version = version.value
+
+    if len(version.split(".")) == 2:
+        version += ".0"
+    if version not in _SUPPORTED_FLUENT_VERSIONS:
```

Comment:
Note that the inability to instantiate a `FluentVersion` indicates the same. i.e. `FluentVersion(version)` raises a ValueError if `version` is invalid. That means that we can eliminate `_SUPPORTED_FLUENT_VERSIONS`. For printing the values, this is the list: [member.value for member in FluentVersion]

Reply:
Thank you @seanpearsonuk, sounds good.

Reply:
Done.